kernel/src/actions/mod.rs

```diff
@@ -660,6 +650,7 @@ mod tests {
     use std::sync::Arc;
 
     use super::*;
+    use crate::actions::schemas::ToSchema;
```

Comment:
```suggestion
    use crate::actions::schemas::ToSchema as _;
```
(since we're not referencing the trait, merely bringing its methods into scope)